integrations/client/test_nowcast.py

```diff
@@ -0,0 +1,133 @@
+"""Integration tests for delphi_epidata.py."""
+
+# standard library
+import unittest
+
+# third party
+import mysql.connector
+
+# first party
+from delphi.epidata.client.delphi_epidata import Epidata
+import delphi.operations.secrets as secrets
+
+# py3tester coverage target
+__test_target__ = 'delphi.epidata.client.delphi_epidata'
+
```

Comment:
nothing changed in this endpoint, right (minus the underlying db that it shares with `covidcast`)? these are just tests for new coverage or am i missing something?